.gitignore

```diff
@@ -1,4 +1,5 @@
 *.pyc
+*.po
```

Comment:
django itself stores both `.po` and `.mo` files. And have no precompication magic in setup.py. I think we should follow the same strategy https://github.com/django/django/tree/master/django/contrib/admin/locale/hu/LC_MESSAGES